app/services/illiad_client.rb

```diff
@@ -51,6 +51,10 @@ def get_loan_holds(webaccess_id)
     ill_get_request(webaccess_id, holds_query)
   end
 
+  def get_user_info(webaccess_id)
+    ill_get_user_request(webaccess_id)
```

Comment:
```suggestion
    ill_response = request("/ILLiadWebPlatform/Users/#{webaccess_id}")
      if ill_response.status == 200
        JSON.parse(ill_response)
      else
        raise JSON.parse(ill_response)['Message']
      end
```

imo there is no need to call a private method here unless we are planning to use `ill_get_user_request` somewhere else as well.